pkg/config/piped.go

```diff
@@ -345,6 +352,13 @@ func (g PipedGit) LoadSSHKey() ([]byte, error) {
 	return nil, errors.New("either sshKeyFile or sshKeyData must be set")
 }
 
+func (g *PipedGit) Validate() error {
+	if g.ShouldConfigureSSHConfig() && g.PersonalAccessToken.ShouldConfigureSSHConfig() {
+		return errors.New("cannot configure both sshKeyData or sshKeyFile and personalAccessToken")
+	}
+	return nil
```

Comment:
Please try not to use `ShouldConfigureSSHConfig`? This function is supposed to be used in a different context. Validate configs here.

Reply:
Sorry... I forgot to fix name when I copied.
How about "ShouldConfigurePATConfig" because the function is checking to use PAT or not?

Reply:
@kentakozuka 
Sorry... I forgot to fix name when I copied.
How about "ShouldConfigurePATConfig" because the function is checking to use PAT or not?